src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -34,7 +34,7 @@ This post explains how to optimize the loading process of third-party scripts wi
 
 ## Use `async` or `defer`
 
-Because [synchronous scripts](link-to-intro) delay DOM construction and rendering, you should always load third-party scripts asynchronously unless the script has to run before the page can be rendered.
+Because [synchronous scripts](https://web.dev/third-party-javascript/) delay DOM construction and rendering, you should always load third-party scripts asynchronously unless the script has to run before the page can be rendered.
```

Comment:
```suggestion
Because [synchronous scripts](/third-party-javascript/) delay DOM construction and rendering, you should always load third-party scripts asynchronously unless the script has to run before the page can be rendered.
```